package.json

```diff
@@ -2,13 +2,15 @@
   "name": "pattern-pedia",
   "version": "0.0.1",
   "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
-    "travis": "ng build --base-href '/pattern-pedia/'"
+    "ng": "node_modules/.bin/ng",
```

Comment:
do we need really need to specify the concrete relative path for every build script? does travis use all of them?

Reply:
Thanks for the feedback. My fault, in case you run the command "manually", it would require a global Angular CLI installation. If yarn is used, it will resolve the path correctly. I removed it in the last commit.